talkativeRobotRGB.rb

```diff
@@ -1,8 +1,29 @@
-puts("How old are you?")
-age = gets().chomp().to_i()
+def askAge
+  puts("How old are you?")
+  gets().chomp().to_i()
+end
+
+def askGender
+  puts("Are you male or female (M/F)?")
+  gets().chomp()
+end
 
-puts("Are you male or female (M/F)?")
-genderInput = gets().chomp()
+def lastMessage(ageDifference,gender)
+
+  if (ageDifference > 60 && gender == "M")
```

Comment:
Although the () is optional, it is clearer to me what the intent is. I am calling the puts method. 

     On Thursday, January 15, 2015 8:20 AM, Gerry Pass <notifications@github.com> wrote:
   

 In talkativeRobotRGB.rb:>  
> -puts("Are you male or female (M/F)?")
> -genderInput = gets().chomp()
> +def lastMessage(ageDifference,gender)
> +
> +  if (ageDifference > 60 && gender == "M")
Don't need the () around any of the if or elsif statements. It adds extra clutter.—
Reply to this email directly or view it on GitHub.